marketdata/rest.go

```diff
@@ -1209,6 +1210,31 @@ func (c *client) GetLatestCryptoXBBO(symbol string, exchanges []string) (*Crypto
 	return &latestXBBOResp.XBBO, nil
 }
 
+// GetCryptoSnapshot returns the snapshot for a given crypto symbol
+func (c *client) GetCryptoSnapshot(symbol string, exchange string) (*CryptoSnapshot, error) {
+	u, err := url.Parse(fmt.Sprintf("%s/%s/%s/snapshot", c.opts.BaseURL, cryptoPrefix, symbol))
+	if err != nil {
+		return nil, err
+	}
+
+	q := u.Query()
+	q.Set("exchange", exchange)
+	u.RawQuery = q.Encode()
+
+	resp, err := c.get(u)
+	if err != nil {
+		return nil, err
+	}
+
+	var snapshot CryptoSnapshot
+
```

Comment:
[superminor] I wouldn't add an empty here.

Reply:
Removed. But honestly, I like these super minor comments so that we keep the code base consistent!